sherpa-onnx/csrc/online-recognizer-transducer-impl.h

```diff
@@ -83,7 +83,8 @@ class OnlineRecognizerTransducerImpl : public OnlineRecognizerImpl {
       : OnlineRecognizerImpl(config),
         config_(config),
         model_(OnlineTransducerModel::Create(config.model_config)),
-        sym_(config.model_config.tokens),
+        sym_(config.model_config.tokens_buf_str.empty() ? config.model_config.tokens :
+             config.model_config.tokens_buf_str, config.model_config.tokens_buf_str.empty() ? true : false),
```

Comment:
Line 86-87 are hard to read.

Could you put the initialization inside `{}`, not in the initializer list?